### ISSUE TITLE
meeting to discuss course divergence

### ISSUE THREAD

**Issue by jsms90** [foundersandcoders/master-reference#510]
Open call for anyone who would like to attend - I'll post on gitter shortly

Agenda:
1. Curriculum planning in London
    + Separation of Curriculum planning and Mentor preparation #501 
    + Week maintainers - think "Product Owner"
      + Who makes updates with them?
      + When do updates get made?
    + Mentor preparation during weeks 10-16
1. Curriculum planning elsewhere (do Nazareth & Gaza need one issue or separate issues?)
1. Course divergence
    + Do we want to enable divergence?
    + How? Potential process for enabling course divergence #511 

Let's confirm a date for next week please @eliascodes @m4v15 @mattlub @rebeccaradding @sofer @des-des @bradreeder 

Additions to the agenda are welcome.

**Comment by rebeccaradding** [comment]
Sometime Sunday or Tuesday would work for me.

**Comment by eliascodes** [comment]
Tuesday/Wednesday are best for me

**Comment by rebeccaradding** [comment]
@jsms90 suggest a time Tuesday? Perhaps touch base with others who are in the space @des-des @bradreeder as well?

**Comment by jsms90** [comment]
I'm thinking the week after might be necessary, to make sure we have attendance from as many people on this list as possible.

My personal availability:
Tuesday: 1pm - 4.30pm
Wednesday: flexible
Thursday: flexible

How is Tuesday 1st August 2-3pm @eliascodes @m4v15 @mattlub @rebeccaradding @sofer @des-des @bradreeder?

**Comment by m4v15** [comment]
Sorry, yeah this week is/was busy, next tuesday should work for me

**Comment by des-des** [comment]
@jsms90 my thoughts have not really changed since I created this document:

https://github.com/foundersandcoders/master-reference/blob/master/vision.md

I still think (very roughly):
  1. Master ref should be a collection of parts (inc weeks)
  2. Campuses each have their own top level document, that composes weeks together.

I was gonna implement this before last sprint planning but blocked by various discussions in issues

**Comment by sofer** [comment]
I am really keen to hear from @majakudlicka @finnhodgkin and @ZooeyMiller on this subject after they have had a chance to deliver Weeks 4 and 5 across all three campuses. I suggest holding off on this until they have reported back.

**Comment by rebeccaradding** [comment]
@jsms90 have opened a new issue #544

**Comment by jsms90** [comment]
Ok :confused: I'll close this then
